internal/newusers/cache/db.go

```diff
@@ -221,34 +200,31 @@ func (c *Cache) Close() error {
 	return c.db.Close()
 }
 
+// Clear removes the database file.
+func Clear(cacheDir string) error {
+	return os.Remove(filepath.Join(cacheDir, dbName))
+}
+
 // ClearAndRebuild closes the db and reopens it.
-func (c *Cache) ClearAndRebuild(cacheDir string) {
+func (c *Cache) ClearAndRebuild(cacheDir string) error {
```

Comment:
and this one then just `Clear`, as in the ends, it’s like we have cleared all entries.